docs/en/changes/changes.md

```diff
@@ -101,6 +101,7 @@
 * Optimize MQ Topology analysis. Use entry span's peer from the consumer side as source service when no producer instrumentation(no cross-process reference).
 * Refactor JDBC storage implementations to reuse logics.
 * Fix `ClassCastException` in `LoggingConfigWatcher`.
+* Support span attached event concept.
```

Comment:
```suggestion
* Support span attached event concept in Zipkin and SkyWalking trace query.
* Support span attached events on Zipkin lens UI.
```

This doc should be updated as well, https://skywalking.apache.org/docs/main/next/en/setup/backend/zipkin-trace/

Also, we should consider adding a menu here to introduce ebpf besides tracing/logging/metrics.
<img width="323" alt="image" src="https://user-images.githubusercontent.com/5441976/200705054-8732349e-21a0-43c6-ab60-1c7a5f90072d.png">

Reply:
Let's update the docs for profiling in the next PR.